src/application/look-up-wallet-address/app.tsx

```diff
@@ -0,0 +1,121 @@
+import { FormEvent, useEffect, useState } from 'react';
+import { useDebounce } from 'react-use';
+
+import { useCommandQuery } from 'shared/messaging';
+import { Closable, IconButton } from 'shared/ui';
+import { useExtensionSettings } from 'shared/extension';
+
+import { GetResolvedAddressCommand, GetTwitterIdsCommand } from './commands';
+import { AddressList } from './address-list';
+
+export const App = () => {
+  const [username, setUsername] = useState<string>('');
+  const [inputValue, setInputValue] = useState<string>('');
```

Comment:
you don't need to pass `<string>` generic, typescript understands type from default value that you've provided